Makefile

```diff
@@ -50,6 +50,7 @@ all: init-block
 .PHONY: build
 build:
 	@echo Building container binaries...
+	@$(SWIFT) --version
```

Comment:
Do you want this on every build or only if CI=true?

Reply:
It's fine on every build - most developers won't even notice it on their local builds but it'll be there if they want a quick check